apps/mudita-center-e2e/src/helpers/screenshot.helper.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import path from "path"
+import { existsSync, mkdirSync } from "fs"
+
+const screenshotsDir = "./screenshots"
+const viewsDir = path.join(screenshotsDir, "views")
+const elementsDir = path.join(screenshotsDir, "elements")
+
+const generateFilename = (baseName: string, elementName?: string) => {
```

Comment:
Move method inside a class and use `this.generateFilename` and variables like this: `this.screenshotDir`, `this.viewsDir`, `this.elementsDir` 

```suggestion
class ScreenshotHelper {
  screenshotsDir = "./screenshots"
  viewsDir = path.join(this.screenshotsDir, "views")
  elementsDir = path.join(this.screenshotsDir, "elements")

  generateFilename = (baseName: string, elementName?: string) => {
    const currentDate = new Date()
    const millisecondsInMinute = 60 * 1000
    const timezoneOffsetMs =
      currentDate.getTimezoneOffset() * millisecondsInMinute
    const dateAsISOString = new Date(
      currentDate.getTime() - timezoneOffsetMs
    ).toISOString()
```